samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -172,6 +174,42 @@ The :c:func:`location_assistance_data_handler` function handles this task, and M
+
+On these board targets, the patterns shown, what states they describe, and what options are required for them to appear are as follows:
+
++------------------------------------------------------+--------------------------------------+----------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| Status                                               | ``thingy91_nrf9160_ns`` (Thingy:91)  | ``nrf9160dk_nrf9160_ns`` (nRF9160 DK)              | Conditions                                                                                                |
++======================================================+======================================+====================================================+===========================================================================================================+
+| Trying to connect to nRF Cloud (for the first time)  | Pulsating orange LED                 | Single lit LED spinning around the square of LEDs  | If LED status indication is enabled                                                                       |
+| Connection to nRF Cloud lost, reconnecting           | Pulsating orange LED                 | Single lit LED spinning around the square of LEDs  | Only if the :ref:`CONFIG_LED_VERBOSE_INDICATION <CONFIG_LED_VERBOSE_INDICATION>` option is enabled        |
+| Fatal Error                                          | Blinking red LED, 75% duty cycle     | All 4 LEDs blinking, 75% duty cycle                | If LED status indication is enabled                                                                       |
+| Device message sent successfully                     | Blinking green LED, 25% duty cycle   | Alternating diagonals blinking                     | Only if the :ref:`CONFIG_LED_VERBOSE_INDICATION <CONFIG_LED_VERBOSE_INDICATION>` option is enabled        |
+| Idle                                                 | LED pulsating between blue and cyan  | Single LED bouncing between opposite corners       | Only if the :ref:`CONFIG_LED_CONTINUOUS_INDICATION <CONFIG_LED_CONTINUOUS_INDICATION>` option is enabled  |
++------------------------------------------------------+--------------------------------------+----------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
+
+Under all other circumstances, onboard LEDs will be turned off.
+
+.. note::
+  The :ref:`CONFIG_LED_VERBOSE_INDICATION <CONFIG_LED_VERBOSE_INDICATION>` and :ref:`CONFIG_LED_CONTINUOUS_INDICATION <CONFIG_LED_CONTINUOUS_INDICATION>` options are enabled by default.
+
+See `nrf_cloud_mqtt_multi_service_customizing_LED_status_indication`_ for details on customizing LED behavior.
+
+See `nrf_cloud_mqtt_multi_service_led_third_party`_ for details on configuring LED status indication on third-party boards.
```

Comment:
```suggestion
See `nrf_cloud_mqtt_multi_service_led_third_party`_ for details on configuring the LED status indication on third-party boards.
```

Reply:
"the" seems a bit unnatural in this sentence to me, "LED status indication" feels like an abstract noun